charts/lnd/templates/export-secrets-configmap.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ printf "%s-export-secrets" (include "lnd.fullname" .) }}
+  labels:
+    {{- include "lnd.labels" . | nindent 4 }}
+data:
+  exportSecrets.sh: |
+    #!/bin/sh
+    kubectl wait --for=condition=ready --timeout=150s pod {{ include "lnd.fullname" . }}-0
+    kubectl cp {{ include "lnd.fullname" . }}-0:/root/.lnd/tls.cert ./tls.cert
+    kubectl create configmap {{ include "lnd.fullname" . }}-tls --from-file=./tls.cert --dry-run=client -o yaml | kubectl apply -f -
```

Comment:
I think it should be a secret, not a configmap

Reply:
Fixed